projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Quotes.php

```diff
@@ -675,10 +675,12 @@ public function html( $quote, $metabox ) {
 
             // localise ID & content
             $quoteID = -1; if (is_array($quote) && isset($quote['id'])) $quoteID = (int)$quote['id'];
-            $quoteContent = ''; if (is_array($quote) && isset($quote['content'])) $quoteContent = $quote['content'];
+			$quote_content = '';
+		if ( is_array( $quote ) && isset( $quote['content'] ) ) {
+			$quote_content = $quote['content'];
+		}
             
-            #http://stackoverflow.com/questions/3493313/how-to-add-wysiwyg-editor-in-wordpress-meta-box
-            $content = zeroBSCRM_io_WPEditor_DBToWPEditor($quoteContent);
+			$content = $quote_content;
```

Comment:
As above, let's use `wp_kses()` here. We also should remove `$content` and just pass `$quote_content` to `wp_editor()`.

Reply:
Good points, thank you.